geometry_msgs/Pose         node_pose
sensor_msgs/PointCloud2    node_sensor